examples/gauge-metrics/gauge_metric.server.out

```diff
@@ -1,44 +1,44 @@
 # To start the service, navigate to the directory that contains the
 # `.bal` file and use the `ballerina run` command with `--observe` param.
 $ ballerina run --observe gauge_metrics.bal
-ballerina: started publishing tracers to Jaeger on localhost:5775
 [ballerina/http] started HTTP/WS Listener 0.0.0.0:9797
-ballerina: started Prometheus HTTP Listener 0.0.0.0:9797
+ballerina: started Prometheus HTTP endpoint 0.0.0.0:9797
```

Comment:
```suggestion
ballerina: started Prometheus HTTP listener 0.0.0.0:9797
```

We have change endpoint to listener and please do the necessary changes here as well.